Prusa-CW-Firmware/main.cpp

```diff
@@ -119,75 +207,50 @@ typedef struct
 } eeprom_t;
 
 static_assert(sizeof(eeprom_t) <= EEPROM_OFFSET, "eeprom_t doesn't fit in it's reserved space in the memory.");
-static constexpr eeprom_t * eeprom_base = reinterpret_cast<eeprom_t*> (E2END + 1 - EEPROM_OFFSET);
-
-
-byte washing_run_time = 4;
-byte curing_run_time = 3;
-byte drying_run_time = 3;
-byte resin_preheat_run_time = 3;
-byte finish_beep_mode = 1;
-byte sound_response = 1;
-byte heat_to_target_temp = 0;
-byte target_temp_celsius = 35;
-byte target_temp_fahrenheit = 95;
-byte resin_target_temp_celsius = 30;
-byte curing_machine_mode;
-byte SI_unit_system = 1;
+static eeprom_t * const eeprom_base = reinterpret_cast<eeprom_t*> (E2END + 1 - EEPROM_OFFSET);
+static eeprom_small_t * const eeprom_small_base = reinterpret_cast<eeprom_small_t*> (E2END + 1 - EEPROM_OFFSET);
+
+eeprom_t config = {"CURW1", 10, 1, 4, 3, 3, 1, 1, 0, 35, 95, 0, 1, false, 60, 60, 40, 70, 70, 40, 3, 30};
 
 byte max_preheat_run_time = 30;
 byte cover_check_enabled = 1;
 byte LED_PWM_VALUE = 100;
 
-int fan1_pwm_State = LOW;
-int fan2_pwm_State = LOW;
+bool fan1_pwm_State = LOW;
+bool fan2_pwm_State = LOW;
```

Comment:
It would be worth to rename to something like fanx_pwm_high and assign everywhere true and false.